From 39d64826c3d93b3e24daebcbc468b369362ed2b3 Mon Sep 17 00:00:00 2001
From: Kristian Evensen <kristian.evensen@gmail.com>
Date: Fri, 24 Feb 2017 14:32:05 +0100
Subject: [PATCH] [SRC] Only allow SSH login from localhost

We should only allow SSH login from localhost (i.e., the reverse tunnel). SSH
access from LAN will be disabled by default, at least until we have properly
figured out how to handle keys, etc.

For development, local SSH access is conventient. Then this patch should be
deleted.
---
 sshd_config | 2 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sshd_config b/sshd_config
index 67a756f..c362703 100644
--- a/sshd_config
+++ b/sshd_config
@@ -12,8 +12,8 @@
 
 #Port 22
 #AddressFamily any
-#ListenAddress 0.0.0.0
-#ListenAddress ::
+ListenAddress 127.0.0.1
+ListenAddress ::1
 
 HostKey /etc/ssh/ssh_host_rsa_key
 HostKey /etc/ssh/ssh_host_ecdsa_key

